notes/src/components/NoteCard.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+import './NoteCard.css';
+
+const NoteCard = ({ title, body, id }) => {
+  return (
+    <Route render={({ history }) => (
```

Comment:
Didn't think of routing this way! I wrapped my card with a `<Link>` tag.

Have you looked into `react-router-redux`? Seems like _the_ way to go because it syncs routing data via the store and allows you to navigate by dispatching actions.

Reply:
I was motivated to learn this way of getting to the history object out of frustration, due to being unable to style the Link tag. The blue link situation was getting me down lol.

> Have you looked into react-router-redux? Seems like the way to go because it syncs routing data via the store and allows you to navigate by dispatching actions.

Absolutely. Since then, this is my favorite way of doing it. The alternative to that or an empty route is making sure you are passing the history object down to your components, but redux just makes everything much easier to reason about after you put in the work.